Makefile

```diff
@@ -0,0 +1,18 @@
+a.out:main.o File.o Directory.o System.o
+	g++ -o a.out main.o Directory.o System.o File.o
+
+main.o: main.cpp
+	g++ -std=c++11 -c main.cpp
+
+File.o: File.cpp
+	g++ -std=c++11 -c File.cpp
+
+Directory.o: Directory.cpp
+	g++ -std=c++11 -c Directory.cpp
+
+System.o: System.cpp
+	g++ -std=c++11 -c System.cpp
```

Comment:
A header fájlokra is szükség van az object fájlok elkészítéséhez, így azokat is fel kell sorolni. (Sőt, igazából a cpp fájlokat el lehet hagyni, mert azokat automatikusan felismeri a `make`, ha azonos nevűek, de maradjanak csak.)

Reply:
Azt pontosan hogyan is kellene?

Reply:
https://github.com/hegyhati/T4MSD/wiki/Make
```
target: dependencies
TAB command
```